donjo-app/models/Keluarga_model.php

```diff
@@ -644,7 +644,7 @@ public function get_kepala_kk($id, $is_no_kk = false)
 
 			$data = $this->db->get()->row_array();
 
-		if ($data['dusun'] != '') $data['alamat_plus_dusun'] = trim($data['alamat'].' '.ucwords($this->setting->sebutan_dusun).' '.$data['dusun']);
+		if ($data['dusun'] != '-' && $data['dusun'] != '') $data['alamat_plus_dusun'] = trim($data['alamat'].' '.ucwords($this->setting->sebutan_dusun).' '.$data['dusun']);
```

Comment:
Logic nya semestinya pakai OR, sehingga isisan '-' atau '' berarti data dusunnya ada,

`if ($data['dusun'] != '-' || $data['dusun'] != '')`

Reply:
iya, kalau pakai `and`, berari dua jika dua kondisi tersebut dipenuhi

Reply:
Dalam hal ini logika @AliAshgar sdh benar, kalau tujuannya adalah untuk mendeteksi tidak ada dusun, di mana tanda '-' digunakan sebagai cara untuk menunjukkan tidak ada dusun.